posthog/models/dashboard_item.py

```diff
@@ -12,7 +12,10 @@
 
 class DashboardItem(models.Model):
     dashboard: models.ForeignKey = models.ForeignKey(
-        "Dashboard", related_name="items", on_delete=models.CASCADE, null=True, blank=True
+        "Dashboard", related_name="items", on_delete=models.CASCADE, null=True, blank=True,
+    )
+    parent: models.ForeignKey = models.ForeignKey(
```

Comment:
I recommend adding some comments here on the code where we document the purpose of this attribute.